packages/core/src/utils/flashFallback.integration.test.ts

```diff
@@ -17,10 +17,13 @@ import {
 import { DEFAULT_GEMINI_FLASH_MODEL } from '../config/models.js';
 import { retryWithBackoff } from './retry.js';
 import { AuthType } from '../core/contentGenerator.js';
+// Import the new types (Assuming this test file is in packages/core/src/utils/)
+import type { FallbackHandler } from '../fallback/types.js';
 
 vi.mock('node:fs');
 
-describe('Flash Fallback Integration', () => {
```

Comment:
not a new issue but for discussion, should this be renamed to flashFallback.test.ts
seems much more like a unit test than a regular integration test with lots of mocking.